django_netjsonconfig/api/urls.py

```diff
@@ -3,6 +3,9 @@
 from . import views
 
 urlpatterns = [
+    url(r'^template/(?P<uuid>[^/]+)/$',
+        views.share_template,
+        name='share_template'),
     url(r'^search/$',
```

Comment:
`/search/` is not very RESTful, I think this should also be named `/templates/`, it should be a list template view with the possibility to filter and search. I did not spot this in the previous PR.